packages/node/src/events/index.ts

```diff
@@ -1,10 +1,11 @@
-export { default as installEventController } from "./install/controller";
+import { installEventController } from "./install/controller";
```

Comment:
No need to import these like this if that file only has one thing being exported (i.e. `export default`)

Reply:
The style of importing things one by one then exporting them all together is nice though.